src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -21,6 +21,8 @@
 import java.util.*;
 
 public final class AlignmentSummaryMetricsCollector extends SAMRecordAndReferenceMultiLevelCollector<AlignmentSummaryMetrics,  Long> {
+    private static final long serialVersionUID = 1L;
```

Comment:
should the class be `Serializable`? right now it has a field `serialVersionUID` but it's not Serializable so the field is useless